docs/getting-started/generated-app-api.md

```diff
@@ -39,7 +39,7 @@ The REST API provide methods on all your data model. For each model, you can fin
 
 The GraphQL API is available at **/graphql** at the root of your application.
 
-When you navigate directly to **/graphql** you will see the [GraphQL Playground](https://www.apollographql.com/docs/apollo-server/testing/graphql-playground/) provided by Apollo Server.
+When you navigate directly to **/graphql** you will see the [GraphQL Playground](https://www.apollographql.com/docs/apollo-server/v2/testing/graphql-playground/) provided by Apollo Server.
```

Comment:
We would probably need to suggest to use Apollo studio sandbox instead of the playground since it is not supported since December 2022